sdk/main/include/TokenCreateTransaction.h

```diff
@@ -0,0 +1,555 @@
+   * characters.
+   */
+  std::string mSymbol;
+
+  /**
+   * For tokens of type FUNGIBLE_COMMON, the number of decimal places by which a token is divisible. This field can
+   * never be changed. For tokens of type NON_FUNGIBLE_UNIQUE, this must be 0.
+   */
+  uint32_t mDecimals = 0U;
+
+  /**
+   * The initial supply of FUNGIBLE_COMMON tokens to be put in circulation. The initial supply is sent to the
+   * Treasury Account. The supply is in the lowest denomination possible. For tokens of type NON_FUNGIBLE_UNIQUE, this
+   * must be 0.
+   */
+  uint64_t mInitialSupply = 0ULL;
+
+  /**
+   * The account which will act as a treasury for the token. This account will receive the specified initial supply or
+   * the newly minted NFTs in the case for NON_FUNGIBLE_UNIQUE type.
+   */
+  std::optional<AccountId> mTreasuryAccountId;
+
+  /**
+   * The Key which can perform update/delete operations on the token. If nullptr, the token can be perceived as
+   * immutable (not being able to be updated/deleted).
+   */
+  std::shared_ptr<Key> mAdminKey = nullptr;
+
+  /**
+   * The Key which can grant or revoke KYC of an account for the token's transactions. If nullptr, KYC is not required,
+   * and KYC grant or revoke operations are not possible.
+   */
+  std::shared_ptr<Key> mKycKey = nullptr;
+
+  /**
+   * The Key which can sign to freeze or unfreeze an account for token transactions. If nullptr, freezing is not
+   * possible.
+   */
+  std::shared_ptr<Key> mFreezeKey = nullptr;
+
+  /**
+   * The Key which can wipe the token balance of an account. If nullptr, wipe is not possible.
+   */
+  std::shared_ptr<Key> mWipeKey = nullptr;
+
+  /**
+   * The Key which can change the supply of a token. The Key is used to sign token mint and burn operations.
+   */
+  std::shared_ptr<Key> mSupplyKey = nullptr;
+
+  /**
+   * The default Freeze status (frozen or unfrozen) of Hedera accounts relative to this token. If true, an account must
+   * be unfrozen before it can receive the token.
+   */
+  bool mFreezeDefault = false;
+
+  /**
+   * The epoch second at which the token should expire. If mAutoRenewAccountId and mAutoRenewPeriod are specified, this
+   * is coerced to the current epoch second plus the mAutoRenewPeriod.
+   */
+  std::chrono::system_clock::time_point mExpirationTime = std::chrono::system_clock::now() + DEFAULT_AUTO_RENEW_PERIOD;
+
+  /**
+   * The ID of the account which will be automatically charged to renew the token's expiration, at the interval
+   * specified in mAutoRenewPeriod.
+   */
+  std::optional<AccountId> mAutoRenewAccountId;
+
+  /**
+   * The interval at which the auto-renew account will be charged to extend the token's expiry.
+   */
+  std::chrono::duration<double> mAutoRenewPeriod = DEFAULT_AUTO_RENEW_PERIOD;
+
+  /**
+   * The memo associated with the token (UTF-8 encoding max 100 bytes).
+   */
+  std::string mMemo;
```

Comment:
Same here - this filed is named [tokenMemo](https://github.com/hashgraph/hedera-sdk-java/blob/274012d54d75c5896414b8406df76a637eb9c58a/sdk/src/main/java/com/hedera/hashgraph/sdk/TokenCreateTransaction.java#L83) in the [Java SDK](https://github.com/hashgraph/hedera-sdk-java).
It's better to use the same naming convention between the C++ and the Java versions.

Reply:
Done.